### PR TITLE
Update electrical magnetic release notes of MSL 4.0.0

### PR DESCRIPTION
**Pull request by christiankral**
Refs #3468

### CODE REVIEW

File: Modelica/package.mo [hunk 1]
```diff
@@ -2451,6 +2454,15 @@ The following <font color=\"blue\"><strong>existing components</strong></font> h
     <td>Added second derivatives and modified Akima interpolation.</td></tr>
 <tr><td>CombiTable2Ds<br>CombiTable2Dv</td>
     <td>Added second derivatives.</td></tr>
+<tr><td colspan=\"2\"><strong>Modelica.Electrical.Analog.Basic</strong></td></tr>
+<tr><td>Gyrator</td>
+    <td>Serves as generalized gyrator model as IdealGyrator was removed.</td></tr>
+<tr><td colspan=\"2\"><strong>Modelica.Electrical.Analog.Ideal</strong></td></tr>
+<tr><td>IdealizedOpAmpLimited</td>
+    <td>Add homotopy to operational amplifier.</td></tr>
```
Comment (beutlich) [inline Modelica/package.mo]:
```suggestion
    <td>Added homotopy to operational amplifier.</td></tr>
```

File: Modelica/package.mo [hunk 2]
```diff
@@ -2451,6 +2454,15 @@ The following <font color=\"blue\"><strong>existing components</strong></font> h
...
+<tr><td colspan=\"2\"><strong>Modelica.Electrical.Semiconductors</strong></td></tr>
+<tr><td>NPN<br>PNP</td>
+    <td>Add optional substrate connector.</td></tr>
```
Comment (beutlich) [inline Modelica/package.mo]:
```suggestion
    <td>Added optional substrate connector.</td></tr>
```

File: Modelica/package.mo [hunk 3]
```diff
@@ -2492,6 +2504,9 @@ that can lead to wrong simulation results):
 <tr><td colspan=\"2\"><strong>Modelica.Blocks.Math</strong></td></tr>
 <tr><td>Pythagoras</td>
     <td>The case with negative <code>y2</code> was not correctly considered if <code>u1IsHypotenuse</code> is <code>true</code>. This has been corrected.</td></tr>
+<tr><td colspan=\"2\"><strong>Modelica.Electrical.Semiconductors</strong></td></tr>
+<tr><td>Diode</td>
+    <td>Fix bug in current equations.</td></tr>
```
Comment (beutlich) [inline Modelica/package.mo]:
```suggestion
    <td>Fixed unit error in current equations.</td></tr>
```

I believe you're refering to https://github.com/modelica/ModelicaStandardLibrary/pull/3010#pullrequestreview-301912956.

Comment (christiankral) [reply Modelica/package.mo]: Yes, particularly https://github.com/modelica/ModelicaStandardLibrary/pull/3010/commits/3f23a7227142ba6a2161aba640a08792f7c7cc9a

**Pull request merged**